src/allotropy/parsers/biorad_bioplex/biorad_bioplex_structure.py

```diff
@@ -0,0 +1,261 @@
+from dataclasses import dataclass, field
+
+from defusedxml import ElementTree as DefusedElementTree
+
+from allotropy.exceptions import AllotropeConversionError
+from allotropy.parsers.utils.values import get_val_from_xml, try_int_or_none
+
+WELLS_TAG = "Wells"
+DOC_LOCATION_TAG = "NativeDocumentLocation"
+DESCRIPTION_TAG = "Description"
+PLATE_DIMENSIONS_TAG = "PlateDimensions"
+TOTAL_WELLS_ATTRIB = "TotalWells"
+VERSION_ATTRIB = "BioPlexManagerVersion"
+SAMPLES = "Samples"
+MEMBER_WELLS = "MemberWells"
+ROW_NUMBER = "RowNo"
+COLUMN_NUMBER = "ColNo"
+WELL_NUMBER = "WellNumber"
+WELL_NO = "WellNo"
+REGION_NUMBER = "RegionNumber"
+REGIONS_OF_INTEREST = "RegionsOfInterest"
+CODE = "Code"
+DILUTION = "Dilution"
+ROW_NAMES = "ABCDEFGH"
+
+SAMPLE_ROLE_TYPE_MAPPING = {
+    "Blank": "blank role",
+    "Control": "control sample role",
+    "Standard": "standard sample role",
+    "Unknown": "unknown sample role",
+}
+
+
+@dataclass(frozen=True)
+class AnalyteSample:
+    analyte_name: str
+    analyte_region: int
+    analyte_error_code: int
+
+    @staticmethod
+    def create(analyte_xml):
+        return AnalyteSample(
+            analyte_name=analyte_xml[0].text,
+            analyte_region=try_int_or_none(analyte_xml.attrib[REGION_NUMBER]),
+            analyte_error_code=try_int_or_none(analyte_xml[1].attrib[CODE]),
+        )
+
+
+@dataclass(frozen=True)
+class WellAnalyteMapping:
+    well_name: str
+    analytes: list[AnalyteSample] = field(default_factory=list)
+
+
+@dataclass
+class SampleDocument:
+    sample_type: str
+    sample_identifier: str
+    description: str
+    well_name: str
+    sample_dilution: float
+    well_analyte_mapping: list[WellAnalyteMapping] = field(default_factory=list)
+
+
+@dataclass
+class SampleDocumentAggregate:
+    # This data class pulled from the <Samples> part of the xml.
+    samples: list[SampleDocument] = field(default_factory=list)
+    # Default to empty dictionary.
+    analyte_region_dict: dict[str, str] = field(default_factory=dict)
+
+    @staticmethod
+    def create(samples_xml):
```

Comment:
Note: When I add the type for samples_xml (DefusedElementTree.Element), I get the following attribute error: AttributeError: module 'defusedxml.ElementTree' has no attribute 'Element'
This was not happening when using the native ElementTree. This looks related: https://github.com/tiran/defusedxml/issues/48.
I could just go back to the native library and add #noqa?